quipucords/api/details_report/util.py

```diff
@@ -13,13 +13,8 @@
     sanitize_row,
 )
 from api.models import (
-    InspectGroup,
-    InspectResult,
-    RawFact,
     Report,
```

Comment:
Trivial nitpick but…
```suggestion
    Report
```
and reformat to lose the parens and line breaks.
(github refused to take a multiline suggestion here 🤷)